core/src/main/cpp/msg/MslControl.cpp

```diff
@@ -563,7 +563,7 @@ void MslControl::init(int32_t numThreads)
         ctx->getRandom()->nextBytes(*keydata);
         const SecretKey encryptionKey(keydata, JcaAlgorithm::AES);
         const SecretKey hmacKey(keydata, JcaAlgorithm::HMAC_SHA256);
-        NULL_MASTER_TOKEN = make_shared<MasterToken>(ctx, Date(), Date(), 1L, 1L, dummy, "dummy", encryptionKey, hmacKey);
+        NULL_MASTER_TOKEN = make_shared<MasterToken>(ctx, Date::now(), Date::now(), 1L, 1L, dummy, "dummy", encryptionKey, hmacKey);
```

Comment:
NULL_MASTER_TOKEN is only used as an object placeholder and is never used. The dates inside don't matter.